packages/eos-components/src/components/CarouselCard.vue

```diff
@@ -98,11 +98,15 @@ export default {
 }
 .card {
   border-radius: $border-radius-lg;
-  padding-left: min(400px, 50%);
+  padding-left: 50%;
   position: relative;
   background-color: $gray-700;
   color: $white;
   border: none;
+
+  @include media-breakpoint-up(lg) {
+    padding-left: 400px;
```

Comment:
Can we also move this hardcoded 400px to the theme?

Reply:
I've moved these values tot he theme styles.